### PR TITLE
changelog v5.0.0

### PR DESCRIPTION
**Pull request by cecipirotto**

### CODE REVIEW

**Review by mtnbarreto — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ All notable changes to PagerTabStrip will be documented in this file.
 - Clean up code by using Swift 5.7 capabilities. 
 - `.pagerTabItem(tag: SelectionType)` modifier now requires tag parameter which identifies the pager tab item. 
 - `pagerTabStripViewStyle(_ style: PagerStyle)` is the new modifier to set up pager style. 
-- Selection value, which indicates wich is the selected page, can be any Hashable value and not just an Int like it was in the previous version.
-- Fix scroll behaivour when pager is added inside another pager or scrllable container. Whenever child page reatch its edges, parent scrollable container scrolls. Basically we added a parameter to disable edge scroll gesture so parent gesture is triggered. 
+- Selection value, which indicates which is the selected page, can be any Hashable value and not just an Int like it was in the previous version.
```
Comment (ValentinaBianco) [inline CHANGELOG.md:13]: The Selection value indicating the selected page now supports any Hashable value, not just an Int as in the previous version.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,15 +10,15 @@ All notable changes to PagerTabStrip will be documented in this file.
...
+- Fix scroll behavior when pager is added inside another pager or scrollable container. Whenever child page reaches its edges, parent scrollable container scrolls. Basically we added a parameter to disable edge scroll gesture so parent gesture is triggered.
```
Comment (ValentinaBianco) [inline CHANGELOG.md:14]: Improved the scroll behavior when the pager is added within another pager or scrollable container. When a child page reaches its edges, the parent scrollable container will scroll. Basically, we added a parameter to disable the edge scroll gesture to trigger the parent gesture.

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,15 +10,15 @@ All notable changes to PagerTabStrip will be documented in this file.
 - Clean up code by using Swift 5.7 capabilities. 
 - `.pagerTabItem(tag: SelectionType)` modifier now requires tag parameter which identifies the pager tab item. 
 - `pagerTabStripViewStyle(_ style: PagerStyle)` is the new modifier to set up pager style. 
-- Selection value, which indicates wich is the selected page, can be any Hashable value and not just an Int like it was in the previous version.
-- Fix scroll behaivour when pager is added inside another pager or scrllable container. Whenever child page reatch its edges, parent scrollable container scrolls. Basically we added a parameter to disable edge scroll gesture so parent gesture is triggered. 
+- Selection value, which indicates which is the selected page, can be any Hashable value and not just an Int like it was in the previous version.
+- Fix scroll behavior when pager is added inside another pager or scrollable container. Whenever child page reaches its edges, parent scrollable container scrolls. Basically we added a parameter to disable edge scroll gesture so parent gesture is triggered. 
 - Added several complex examples showing new functionality.
```
Comment (ValentinaBianco) [inline CHANGELOG.md:15]: Added several complex examples to showcase new functionality.

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,15 +10,15 @@ All notable changes to PagerTabStrip will be documented in this file.
...
 - `.pagerTabItem(tag: SelectionType)` modifier now requires tag parameter which identifies the pager tab item. 
 - `pagerTabStripViewStyle(_ style: PagerStyle)` is the new modifier to set up pager style. 
-- Selection value, which indicates wich is the selected page, can be any Hashable value and not just an Int like it was in the previous version.
-- Fix scroll behaivour when pager is added inside another pager or scrllable container. Whenever child page reatch its edges, parent scrollable container scrolls. Basically we added a parameter to disable edge scroll gesture so parent gesture is triggered. 
+- Selection value, which indicates which is the selected page, can be any Hashable value and not just an Int like it was in the previous version.
+- Fix scroll behavior when pager is added inside another pager or scrollable container. Whenever child page reaches its edges, parent scrollable container scrolls. Basically we added a parameter to disable edge scroll gesture so parent gesture is triggered. 
 - Added several complex examples showing new functionality. 
 - Scrollable style page now uses iOS 16 native layout engine through Layout protocol.
```
Comment (ValentinaBianco) [inline CHANGELOG.md:16]: The scrollable style page now utilizes the iOS 16 native layout engine through the Layout protocol.

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,15 +10,15 @@ All notable changes to PagerTabStrip will be documented in this file.
...
-- Fixed inicator initial animation and position issue when presenting the page. 
-- Fixed pages reording, adding, deleting. Now it works perfectly. 
-- `public func onPageAppear(perform action: (() -> Void)?) -> some View` was removed. You should use onAppear native calback or useg selection state variable.
+- Fixed indicator initial animation and position issue when presenting the page.
```
Comment (ValentinaBianco) [inline CHANGELOG.md:17]: Fixed the initial animation and position issue of the indicator when presenting the page.

File: CHANGELOG.md [hunk 6]
```diff
@@ -10,15 +10,15 @@ All notable changes to PagerTabStrip will be documented in this file.
...
+- Selection value, which indicates which is the selected page, can be any Hashable value and not just an Int like it was in the previous version.
+- Fix scroll behavior when pager is added inside another pager or scrollable container. Whenever child page reaches its edges, parent scrollable container scrolls. Basically we added a parameter to disable edge scroll gesture so parent gesture is triggered. 
 - Added several complex examples showing new functionality. 
 - Scrollable style page now uses iOS 16 native layout engine through Layout protocol.
-- Fixed inicator initial animation and position issue when presenting the page. 
-- Fixed pages reording, adding, deleting. Now it works perfectly. 
-- `public func onPageAppear(perform action: (() -> Void)?) -> some View` was removed. You should use onAppear native calback or useg selection state variable.
+- Fixed indicator initial animation and position issue when presenting the page. 
+- Fixed errors on reordering, adding and deleting pages. Now it works perfectly.
```
Comment (ValentinaBianco) [inline CHANGELOG.md:18]: Resolved errors when reordering, adding, and deleting pages. Now it works perfectly.

File: CHANGELOG.md [hunk 7]
```diff
@@ -10,15 +10,15 @@ All notable changes to PagerTabStrip will be documented in this file.
...
+- Fix scroll behavior when pager is added inside another pager or scrollable container. Whenever child page reaches its edges, parent scrollable container scrolls. Basically we added a parameter to disable edge scroll gesture so parent gesture is triggered. 
 - Added several complex examples showing new functionality. 
 - Scrollable style page now uses iOS 16 native layout engine through Layout protocol.
-- Fixed inicator initial animation and position issue when presenting the page. 
-- Fixed pages reording, adding, deleting. Now it works perfectly. 
-- `public func onPageAppear(perform action: (() -> Void)?) -> some View` was removed. You should use onAppear native calback or useg selection state variable.
+- Fixed indicator initial animation and position issue when presenting the page. 
+- Fixed errors on reordering, adding and deleting pages. Now it works perfectly. 
+- `public func onPageAppear(perform action: (() -> Void)?) -> some View` was removed. You should use onAppear native callback or use selection state variable.
 - Refactor enable disable swipe gesture. It's a Binding parameter from now.
```
Comment (ValentinaBianco) [inline CHANGELOG.md:20]: Refactored the enable/disable swipe gesture. Now it's a Binding parameter.

**Review by ValentinaBianco — COMMENTED** [review]
Copy changes

**Pull request merged**